client/dist/styles.css

```diff
@@ -1,7 +1,6 @@
 #app {
   background-color: lightgray;
   display: flex;
```

Comment:
Adding `flex-direction: column` to #app will fixes the pseudo-horizontal layout that we have now

Reply:
I'll definitely try that!